plugins/org.yakindu.sct.simulation.ui/plugin.xml

```diff
@@ -24,7 +24,7 @@
           allowMultiple="false"
           class="org.yakindu.sct.simulation.ui.view.SimulationView"
           icon="icons/Statechart-Launcher-16.png"
-          id="org.yakindu.sct.simulation.ui.declarationview"
```

Comment:
Even if the new id is better we can't change the view's ID since this will cause errors when existing workspaces are opened

Reply:
Changed back the view's ID